Develop

ayaderiah opened the pull request.

ayaderiah committed:
Add index.html with 'Hello world!' text

ayaderiah committed:
feat(index.html): add initial content with 'Hello world!' text

ayaderiah committed:
feat(index.html): change text color to yellow

ayaderiah committed:
feat(index.html): change text color to yellow

ayaderiah committed:
Change text color to yellow in index.html

MKhasib commented:
@ayaderiah 
Please read this:
https://www.conventionalcommits.org/en/v1.0.0/
and read:
- how to update a commit message(or amend)
- how to squash multiple commit messages, try to create 1 commit per PR, try to aim to create small PRs unless you need big one, only then you can create multiple commits if surves the purpose of the PR title(watch on youtube how to do an interactive rebase to combine multiple commits into one)
- how to push force(to update remote changes,please use carefully as it has destructive behavior)

ayaderiah committed:
chore(develop): merge remote updates

Merged changes from the remote develop branch:
https://github.com/ayaderiah/Git-essentials
